packages/design-system/src/styles/components/_UsaBanner.scss

```diff
@@ -60,37 +68,44 @@
   }
```

Comment:
I think we need to add a `padding-right: 2px` for `ds-c-usa-banner__button::after` as well to prevent the icon from cutting off